platforms/documentation/docs/src/snippets/multiproject/buildSrc/groovy/shared/build.gradle

```diff
@@ -0,0 +1,38 @@
+// tag::non-common-logic[]
+plugins {
+    id("java-common-conventions")
+}
+
+dependencies {
+    implementation("com.google.guava:guava:32.1.2-jre")
+}
+// end::non-common-logic[]
+
+/*
```

Comment:
Is it intentional to have lines 11-38 commented out? Here and in other files. I wish I could see the final rendering, maybe the tag::common-logic is making it clearer to users this logic is coming from buildSrc?

Reply:
It was intentional. I separated it into a new "no-BuildSrc" snippet instead so its cleaner.